Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -10,6 +10,22 @@ public static class ArrayOperations
         public static void Sort(int[] array)
         {
             // ToDo: implement.
+            if(array.Length > 1 || array != null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this condition so that it does not always evaluate to 'true'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4940124)